### DISCUSSION TITLE
TipTap integration

### DISCUSSION THREAD

**Discussion by macmillen** [streamich/json-joy#693] [Q&A]
I saw that there is already an integration for the Quill-Editor. Is there also going to be one for [TipTap](https://tiptap.dev/)?

**Comment by etclub** [comment]
Or prosemirror

**Comment by streamich** [comment]
I've investigated this topic and now think that a correct high-quality integration with Prosemirror, Quill and Lexical are possible. It will be implemented if funding for this work will be secured.
